packages/core/README.md

```diff
@@ -80,6 +80,56 @@ The mapConfiguration allows controlling many client instance details.
 | extendedMasterportalapiMarkers | extendedMasterportalapiMarkers? | Optional. If set, all configured visible vector layers' features can be hovered and selected by mouseover and click respectively. They are available as features in the store. Layers with `clusterDistance` will be clustered to a multi-marker that supports the same features. Please mind that this only works properly if you configure nothing but point marker vector layers styled by the masterportalAPI. |
 | stylePath | string? | If no link tag with `data-polar="true"` is found in the document, this path will be used to create the link node in the client itself. It defaults to `'./style.css'`. Please mind that `data-polar="true"` is deprecated since it potentially led to flashes of misstyled content. stylePath will replace that solution in the next major release. |
 | renderFaToLightDom | boolean? | POLAR requires FontAwesome in the Light/Root DOM due to an [unfixed bug in many browsers](https://bugs.chromium.org/p/chromium/issues/detail?id=336876). This value defaults to `true`. POLAR will, by default, just add the required CSS by itself. Should you have a version of Fontawesome already included, you can try to set this to `false` to check whether the versions are interoperable. |
+| checkServiceAvailability | boolean? | If set to `true`, all services' availability will be checked with head requests. |
+
+For more information about the layerConf parameter, click [here](#mapconfigurationlayerconf)
```

Comment:
Continuation of https://github.com/Dataport/polar/pull/158/files#r1722300678

I think simply writing "for more in information, see the relevant chapter" on the comment of layerConf is enough. This just leads to a third place where a person should look (without any realy extra info)

Reply:
7e28c3344cdc41ef8f9663910a2954078be0eb0c